src/tests/unit/mock-helpers/mock-module-helpers.tsx

```diff
@@ -55,10 +55,15 @@ export function useOriginalReactElements(library: string, components: any[]) {
 
 function mockReactComponent<T extends React.ComponentClass<P>, P = any>(component, elementName?) {
     if (component !== undefined) {
-        const name =
-            elementName || component.displayName
-                ? `mock-${component.displayName}`
+        let name;
```

Comment:
made this change because in react18 the object structure of component is coming as different, so accordingly updated the helper